rogue-thi-app/components/cards/FoodCard.jsx

```diff
@@ -21,11 +21,19 @@ export default function FoodCard () {
   const [foodError, setFoodError] = useState(null)
   const {
     selectedRestaurants,
+    selectedLanguageFood,
     preferencesSelection,
     allergenSelection
   } = useContext(FoodFilterContext)
   const { i18n, t } = useTranslation(['dashboard', 'food'])
 
+  let languageFood = ''
```

Comment:
This can be simplified to:
```js
  const languageFood = selectedLanguageFood && selectedLanguageFood !== 'default' ? selectedLanguageFood : i18n.languages[0]
```

Reply:
I do not think the `&&` and `? : ` operators should be mixed without braces, as the precendence might not be clear to the reader